pom.xml

```diff
@@ -1848,5 +1858,142 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>quick</id>
+            <activation>
+                <property>
+                    <name>quick</name>
+                </property>
+            </activation>
+            <properties>
+                <skipTests>true</skipTests>
+                <maven.test.skip>true</maven.test.skip>
+                <maven.site.skip>true</maven.site.skip>
+                <maven.source.skip>true</maven.source.skip>
+                <maven.javadoc.skip>true</maven.javadoc.skip>
+                <air.check.skip-all>true</air.check.skip-all>
+            </properties>
+            <build>
+                <plugins>
+                    <!-- Disable plugins entirely, instead of setting the skip option -->
```

Comment:
What's time-delta between skip-all vs disabling plugins entirely? We need to keep this list in sync with any new checks that get added in airbase and it's quite verbose.

Reply:
I should read more. It's 40 seconds without mvnd. 10 seconds with mvnd.

Reply:
The 40 seconds difference is most likely caused by skipping test compilation, not just disabling plugins. I ran this once now and the difference gets smaller, around 10 seconds, for only disabling plugins, and since I'm running this on my MacBook the error here might be pretty high, so I'd need to run this multiple times and it's 4 minutes each time, so it'll take me a while.

Reply:
Anyway, I'll look into adding this to airbase, and measure again.